blocks/library/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+.blocks-quote {
+	margin: 0;
+	box-shadow: inset 0px 0px 0px 0px $light-gray-500;
+	transition: all .2s ease;
```

Comment:
What's the effect we're trying to achieve with the transition? At least in my browser, the border width effect feels a little janky when switching between styles.